### DISCUSSION TITLE
Modularity of the degree

### DISCUSSION THREAD

**Discussion by MakisH** [the-teachingRSE-project/RSE-Masters#40] [Ideas]
A discussion point is whether this should be a full curriculum, or rather modular. Some ideas discussed:

- Define a core of the curriculum, allowing universities to deviate on the electives
- Have micro-degrees (idea by B. Rümpe)
- Have it as a cross-university add-on degree, like [BGCE](https://www.bgce.de/)

**edit (June 2025):** This issues was only meant to document ideas discussed in a teachingRSE meeting in May 2024. It does not originate from the KTS-funded project.

**Comment by annalenalamprecht** [comment]
The idea of the KTS-funded project is definitely to define a reference curriculum (i.e. a clear definition of the core that guarantees some baseline, but also with flexibility for the implementing organizations to tailor it to their situation). 

Micro-degrees is a complementary idea, I would say. 

Cross-university is a really nice idea. However, probably that is not the point of the reference curriculum, but more a practical issue. I mean, if some universities agree that they want to offer this together, they can do it based on the curriculum, but it should also work for one organization alone.

**Comment by juliandehne** [comment]
I prepared a presentation [here](https://github.com/the-teachingRSE-project/RSE-Masters/blob/tschira-funding/presentation/presentation.qmd) discussing some of the decision points. 

One of the major ones that we only discussed in the deRSE group was the idea of profiles (branches, versions) of the Curriculum. The GI Curriculum for Data Science for instance created three branches depending on the student bachelor degree. 

In the presentation I argued for  profiles with:

- at least CS-specialization idea and domain-RSE (i.e. computational social scientist) are competing
- universities offer different existing modules: easier adaptability
- the curriculum can be adapted to other educational systems
- the curriculum can be adapted to other institutions such as universities of applied sciences

**Comment by juliandehne** [comment]
As the Github Action is still broken for the presentation here the hardwired current version [presentation.pdf](https://github.com/user-attachments/files/20705069/presentation.pdf)
